keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -61,15 +61,4 @@ class AdminController(
   ) {
     adminService.forceConstraintReevaluation(application, environment, type)
   }
-
-  //This endpoint will update old artifact version records with their metadata, if exists, by type [deb/docker/npm].
-  //Please note: this is an admin endpoint and is not intent to be used more than once per environment for now.
-  @PostMapping(
-    path = ["/artifacts/{type}/metadata/backfill"]
```

Comment:
No longer used.

Reply:
why? I was going to refactor and reuse it in another way.

Reply:
As I mentioned in a related comment (https://github.com/spinnaker/keel/pull/1530#discussion_r493838264), my understanding was that we'd no longer use this endpoint. If we decide to reintroduce (parts of) this refactoring, I'll restore it.